pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/applicationconfiguration_without_traitdefinition_test.go

```diff
@@ -83,9 +83,6 @@ spec:
       containers:
         - name: nginx
           image: nginx:1.9.4
-          ports:
-            - containerPort: 80
-              name: nginx
```

Comment:
why remove it?

Reply:
server-side apply complaints for missing protocol field in ports. and it seems ports field has no impact on this case.

Reply:
I think this has something to do with CRD and K8s version

Reply:
The reason why SSA complaints missing protocol field is, start from k8s v1.16, new Go struct markers, `//+listType` and `//+listMapKey`, are introduced. For `type ContainerPort struct`, [SSA requires `containerPort` and `protocol` as associative list's map key](https://github.com/kubernetes/kubernetes/blob/5e22f7feadeb97aa8fad9330c936104479ad56d6/staging/src/k8s.io/api/core/v1/types.go#L2236-L2238). Even though `protocol` field has default value "TCP", but SSA works before defaulting system assign default value. So if `protocol` is missed, SSA will fail.

I will revert this deletion and complete them.